modules/multi-architecture-scheduling-examples.adoc

```diff
@@ -0,0 +1,122 @@
+
+= Sample multi-architecture node workload deployments 
+
+There are various cases to consider when you are scheduling workloads on a cluster with compute nodes of different architectures. 
+
+Using node affinity to schedule workloads on a node:: To allow a workload to be scheduled on a certain architecture supported by its images, you can set the `spec.affinity.nodeAffinity` field in your pod specification deployment.
++
+.Example deployment with the `nodeAffinity` set to certain architectures 
+[source,yaml]
+----
+apiVersion: apps/v1
+kind: Deployment
+metadata: [...]
+spec:
+  [...]
+  template:
+    [...]
+    spec:
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+            - matchExpressions:
+              - key: kubernetes.io/arch
+                operator: In
+                values: <1>
+                - amd64
+                - arm64
+----
+<1> Specify the set of supported architecture. Valid values include `amd64` and `arm64`.
+
+Tainting nodes for a specific architecture:: You can taint a node to avoid workloads that are not compatible with an architecture to be scheduled on that node. In clusters using a `MachineSet` object, you can add parameters to a `.spec.template.spec.taints` field to avoid workloads being scheduled on nodes with non-supported architectures. 
++
+.Example `MachineSet` with a taint set
+[source,yaml]
+----
+apiVersion: machine.openshift.io/v1beta1
+kind: MachineSet
+metadata: [...]
+spec:
+  [...]
+  template:
+    [...]
+    spec:
+      [...]
+      taints:
+      - effect: NoSchedule         
+        key: multi-arch.openshift.io/arch                
+        value: arm64
+----
+You can also set a taint on a specific node by running the following command: 
++
+[source,terminal]
+----
+$ oc adm taint nodes <node-name> multi-arch.openshift.io/arch=arm64:NoSchedule
+----
+
+Creating a default toleration:: You can annotate a namespace so all of the workloads get the same default toleration by running the following command:
++
+[source,terminal]
+----
+$ oc annotate namespace my-namespace \
+  'scheduler.alpha.kubernetes.io/defaultTolerations'='[{"operator": "Exists", "effect": "NoSchedule", "key": "multi-arch.openshift.io/arch"}]'
+----
+
+Setting a toleration on a tainted node:: On a node with a defined taint, workloads will not be scheduled on that node. However, you can allow them to be scheduled by setting a toleration in a deployment.
+
++
+.Example deployment with a toleration 
+[source,yaml]
+----
+apiVersion: apps/v1
+kind: Deployment
+metadata: [...]
+spec:
+  [...]
+  template:
+    [...]
+    spec:
+      tolerations:
+      - key: "multi-arch.openshift.io/arch"
+        value: "arm64"
+        operator: "Equal"
+        effect: "NoSchedule"
+----
+This deployment can be used when you have an existing workload image that is not compatible with the architecture of a new node. 
```

Comment:
The example deployment will be allowed on nodes with the `multi-arch.openshift.io/arch=arm64` taint